tendrl/commons/utils/ssh/authorize_key.py

```diff
@@ -0,0 +1,47 @@
+from ansible_module_runner import AnsibleExecutableGenerationFailed
+from ansible_module_runner import AnsibleRunner
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+ANSIBLE_MODULE_PATH = "core/system/authorized_key.py"
+
+
+class AuthorizeKey(object):
```

Comment:
Determine the output data structure for this utility. Make sure that all return values use the same data structure. Write error messages into logs at warning priority.